### ISSUE TITLE
RPN loss function

### ISSUE THREAD

**Issue by groot-1313** [yhenon/keras-frcnn#212]
Hi, I have a question regarding the format of the the labels to calculate the rpn_regr_loss

    def rpn_loss_regr(num_anchors):
	    def rpn_loss_regr_fixed_num(y_true, y_pred):
		if K.image_dim_ordering() == 'th':
			x = y_true[:, 4 * num_anchors:, :, :] - y_pred
			x_abs = K.abs(x)
			x_bool = K.less_equal(x_abs, 1.0)
			return lambda_rpn_regr * K.sum(
				y_true[:, :4 * num_anchors, :, :] * (x_bool * (0.5 * x * x) + (1 - x_bool) * (x_abs - 0.5))) / K.sum(epsilon + y_true[:, :4 * num_anchors, :, :])
		else:
			x = y_true[:, :, :, 4 * num_anchors:] - y_pred
			x_abs = K.abs(x)
			x_bool = K.cast(K.less_equal(x_abs, 1.0), tf.float32)

			return lambda_rpn_regr * K.sum(
				y_true[:, :, :, :4 * num_anchors] * (x_bool * (0.5 * x * x) + (1 - x_bool) * (x_abs - 0.5))) / K.sum(epsilon + y_true[:, :, :, :4 * num_anchors])

	    return rpn_loss_regr_fixed_num

Speaking in terms of tensorflow backend, y_true has 4 dimensions:
If input size was 1000x600
1. batch_size
2. Number of rows ~ 40
3. Number of columns ~ 60
4. 4*Number of anchors - ??

I am training just an RPN on my own dataset. I am unsure how to modify my labes/bounding boxes to be given as input to the loss function. 
From what I understand my y_true needs to be of size 60x40, so will I be required to create outputs of that size with bounding boxes scaled down too?
